src/commands/misc_commands/news.ts

```diff
@@ -0,0 +1,443 @@
+    sendInfoMessage,
+} from "../../helpers/discord_utils";
+import Eris from "eris";
+import LocaleType from "../../enums/locale_type";
+import MessageContext from "../../structures/message_context";
+import NewsRange from "../../enums/news_range";
+import State from "../../state";
+import dbContext from "../../database_context";
+import i18n from "../../helpers/localization_manager";
+import schedule from "node-schedule";
+import type { DefaultSlashCommand } from "../interfaces/base_command";
+import type BaseCommand from "../interfaces/base_command";
+import type CommandArgs from "../../interfaces/command_args";
+import type HelpDocumentation from "../../interfaces/help";
+import type NewsSubscription from "../../interfaces/news_subscription";
+
+enum Action {
+    SUBSCRIBE = "subscribe",
+    UNSUBSCRIBE = "unsubscribe",
+    GET = "get",
+}
+
+const RANGE_OPTION = "range";
+
+const COMMAND_NAME = "news";
+const logger = new IPCLogger(COMMAND_NAME);
+
+const scheduledJobName = (
+    guildID: string,
+    textChannelID: string,
+    range: NewsRange,
+): string => `${guildID}-${textChannelID}-${range}`;
+
+export default class NewsCommand implements BaseCommand {
+    help = (guildID: string): HelpDocumentation => ({
+        name: COMMAND_NAME,
+        description: i18n.translate(guildID, "command.news.help.description"),
+        examples: [
+            {
+                example: `${clickableSlashCommand(COMMAND_NAME)} ${
+                    Action.GET
+                } ${RANGE_OPTION}:${NewsRange.DAY}`,
+                explanation: i18n.translate(
+                    guildID,
+                    "command.news.help.example.get",
+                ),
+            },
+            {
+                example: `${clickableSlashCommand(COMMAND_NAME)} ${
+                    Action.SUBSCRIBE
+                } ${RANGE_OPTION}:${NewsRange.WEEK}`,
+                explanation: i18n.translate(
+                    guildID,
+                    "command.news.help.example.subscribe",
+                ),
+            },
+            {
+                example: `${clickableSlashCommand(COMMAND_NAME)} ${
+                    Action.UNSUBSCRIBE
+                } ${RANGE_OPTION}:${NewsRange.DAY}`,
+                explanation: i18n.translate(
+                    guildID,
+                    "command.news.help.example.unsubscribe",
+                ),
+            },
+        ],
+        priority: 500,
+    });
+
+    call = async ({ message }: CommandArgs): Promise<void> => {
+        logger.warn("Text-based command not supported for news");
+        await sendDeprecatedTextCommandMessage(
+            MessageContext.fromMessage(message),
+        );
+    };
+
+    slashCommands = (): Array<
+        DefaultSlashCommand | Eris.ChatInputApplicationCommandStructure
+    > => [
+        {
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: Action.GET,
```

Comment:
`/news get` isnt an intuitive command for the user, especially for a command that might be used frequently. we can just do `/news [daily/weekly/month]`

Reply:
I don't think it's possible to mix a regular command with subcommands, is there another word to replace `get` with?